internal/test/test.go

```diff
@@ -617,3 +617,10 @@ func (h *Helper) GetCommit(repo string) string {
 	}
 	return strings.TrimSpace(string(out))
 }
+
+func (h *Helper) RemoveFile(path string) {
+	err := os.RemoveAll(path)
```

Comment:
Ah yeah, I see where it's used.

Y'know, let's actually not have this helper, for the same reason we avoid similar ones elsewhere. Putting the error into a helper func obscures the actual spot where it was called from (even though there's only one place right now), which makes it harder to debug tests.

Let's just call `os.RemoveAll()` directly in the test, and if an error occurs, then `t.Fatal(err)` is sufficient.